generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -68,7 +68,9 @@ quarkus.swagger-ui.always-include=true
 quarkus.swagger-ui.path=/swagger-ui
 
 quarkus.native.additional-build-args=\
+<%_ if (authenticationType !== 'oauth2') { _%>
```

Comment:
Given the fact `resources-config.json` is specific to JWT (basically it explicit the need to include the privateKey for token signature) could you use the condition`authenticationType === 'jwt'`
please